packages/easysearch:core/lib/core/search-collection.js

```diff
@@ -192,7 +192,7 @@ class SearchCollection {
           () => this.changed(
             collectionName,
             'searchCount' + definitionString,
-            { count: cursor.mongoCursor.count && cursor.mongoCursor.count() || 0 }
+            { count: cursor.count && cursor.count() || 0 }
```

Comment:
I don't think this is a correct solution, as the mongoCursor has the count based on the current `skip` and `limit`, see https://github.com/matteodem/meteor-easy-search/blob/65d883a1b69ee2d9b8e1f0a55987bfb31c7727fb/packages/easysearch:core/lib/engines/mongo-db.js#L99